Travis CI failure looks legit:
```
# github.com/cilium/cilium/pkg/azure/api

vet: pkg/azure/api/api_interaction_test.go:60:60: too few arguments in call to client.GetInstances

Makefile:395: recipe for target 'govet' failed

make[1]: *** [govet] Error 2

make[1]: Leaving directory '/home/travis/gopath/src/github.com/cilium/cilium'

Makefile:217: recipe for target 'unit-tests' failed

make: *** [unit-tests] Error 2
```

--- a/pkg/k8s/apis/cilium.io/v2/client/register_test.go
+++ b/pkg/k8s/apis/cilium.io/v2/client/register_test.go
@@ -18,11 +18,15 @@ package client
 
 import (
 	"regexp"
+	"testing"
 
 	. "gopkg.in/check.v1"
 	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { TestingT(t) }
 
 type CiliumV2RegisterSuite struct{}
 

--- a/pkg/service/store/store_test.go
+++ b/pkg/service/store/store_test.go
@@ -1,4 +1,4 @@
-// Copyright 2018 Authors of Cilium
+// Copyright 2018-2020 Authors of Cilium
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
@@ -17,11 +17,16 @@
 package store
 
 import (
+	"testing"
+
 	"github.com/cilium/cilium/pkg/checker"
 	"github.com/cilium/cilium/pkg/loadbalancer"
 
 	"gopkg.in/check.v1"
 )
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { check.TestingT(t) }
 
 type ServiceGenericSuite struct{}
 

